src/simmate/apps/badelf/toolkit/grid.py

```diff
@@ -9,10 +9,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 from numpy.typing import ArrayLike
-from pybader.interface import Bader
 from pymatgen.io.vasp import VolumetricData
 from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
-from pyrho.pgrid import PGrid
```

Comment:
why is `toolkit.Grid` in `badelf` instead of `bader`? To me, having `bader` depend on `badelf` seems backwards, You can still leave these imports "hidden", and list them as optional dependencies for the bader app, but required for the badelf.

Or do you think this is more complicated?